pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "excalibur-tests"
+version = "0.0.1"
+authors = [
+    {name = "Mose Giordano", email = "m.giordano@ucl.ac.uk"},
+    {name = "Tuomas Koskela", email = "t.koskela@ucl.ac.uk"},
+    {name = "Ilektra Christidi", email = "ilektra.christidi@ucl.ac.uk"},
+    {name = "Tom Young", email = "t.young@ucl.ac.uk"}
+]
+description = "Framework for building and running HPC benchmarks"
+readme = "README.md"
+requires-python = ">=3.6"
+keywords = [
+    "benchmark",
+    "hpc"
+]
+license = {text = "Apache-2.0"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "reframe-hpc >= 3.11.0, < 4.0.0",
+    "matplotlib >= 3.0.0",
+]
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["modules","apps","examples","envs","perflogs"]
```

Comment:
I would suggest putting all these directories under a directory name, that then is the same as the package name. So people can do `import excalibur_test`  or internally makes it easier (e.g, examples) to load it.
It will also make that line easier by including that directory.